app/components/GuideReviewForm/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Select from 'components/Select';
+
+import PageSection from 'components/PageSection';
+import BorderBottomDiv from 'components/shared/BorderBottomDiv';
+import Stars from 'components/Stars';
+
+class GuideReviewForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
```

Comment:
Let's define out functions like `handleButtonClick = (e) => {` instead of `handleButtonClick(e) {`, and then we can remove these weird bind things in the constructor

Reply:
Sounds good. I found having to bind everything annoying, but I was copying the patterns already present in the project.